src/components/textarea/bl-textarea.css

```diff
@@ -0,0 +1,131 @@
+:host {
+  display: inline-block;
+  width: 200px;
+  position: relative;
+
+  --bl-textarea-padding-vertical: var(--bl-size-2xs);
+  --bl-textarea-padding-horizontal: var(--bl-size-xs);
+  --bl-textarea-border-color: var(--bl-color-border);
+  --bl-textarea-icon-color: var(--bl-color-content-tertiary);
+  --bl-textarea-text-color: var(--bl-color-content-primary);
+  --bl-input-height: var(--bl-size-2xl);
+}
+
+textarea {
+  outline: none;
+  box-sizing: border-box;
+  border: solid 1px var(--bl-textarea-border-color);
```

Comment:
I don't know if we discussed this but component should have a `:focus` or `:focus-within` style for keyboard navigation.